.git-blame-ignore-revs

```diff
@@ -0,0 +1,8 @@
+# Reformat with clang-format-15
+423a4667c203a60f5f6eb2c82dba0f829129b137
```

Comment:
I guess you'll not squash merge this PR? This commit already seems to be out of date.

Reply:
Please read the first comment of the PR...

Reply:
Sorry about that.

I'm still leaving this comment unresolved though until rebasing and the hash has been updated.